pymc/tests/test_distributions_moments.py

```diff
@@ -385,6 +386,26 @@ def test_gamma_moment(alpha, beta, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
+    "alpha, beta, size, expected",
+    [
+        (5, 1, None, 1 / 4),
+        (0.5, 1, None, 1 / 1.5),
+        (1, 1, None, 0.5),
+        (5, 1, 5, np.full(5, 1 / (5 - 1))),
+        (np.arange(2, 7), 1, None, 1 / np.arange(1, 6)),
+        (np.arange(2, 7), 1, 5, 1 / np.arange(1, 6)),
+        (2, np.arange(1, 6), None, np.arange(1, 6)),
+        (np.arange(2, 7), np.arange(1, 6), 5, np.ones(5)),
+        (np.arange(1, 6), 1, None, np.array([0.5, 1, 1 / 2, 1 / 3, 1 / 4])),
+    ],
+)
+def test_inverse_gamma_moment(alpha, beta, size, expected):
```

Comment:
Can we reduce a bit the number of tests?

Reply:
Better now?